eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -309,36 +309,33 @@ stages:
     ### Job for ultralite build ###
```

Comment:
I thought ultralite is only a thing for source-only?

Reply:
yeah I left it here just to comply with the "ultralite" convention but we only ever pass "lite" or "full" for unified build

Reply:
Yeah I would remove the comment and merge ultralite with lite. It's at least confusing to me.